public/403.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>The page you were looking for doesn't exist (403)</title>
```

Comment:
The title is still incorrect. :)

Reply:
Big facepalm moment

Reply:
Hey @lokisk1155 -- If I'm reading this correctly, this small adjustment is still outstanding.  Can you address it?

Reply:
Hi Cielf, yes sorry I’ve been inactive for awhile but I will try to finish all my open pr’s this week! This one I can do in a bit 

Been super busy irl past few months